src/CheckOut.js

```diff
@@ -308,20 +307,22 @@ class CheckOut extends React.Component {
     const parsed = getCheckoutSettings(settings.records);
 
     if (!parsed.checkoutTimeout) {
-      this.timer = null; // so we don't keep trying
+      window.checkOutSessionEndTimer = null; // so we don't keep trying
       return;
```

Comment:
We can only get here if this value evaluated to falsey on line 300. Do we really need to set it to `null` before we return? I realize you're just following along with the existing implementation and it's fine with me if you choose to leave it as is. Still, I wish there a comment here explaining what makes it necessary, so we could be more confident that it _is_ necessary, or if it's not because (e.g.) it's cruft leftover from before the `if (timer)` condition was added up on line 300.

Reply:
Seems that you are right. I will remove it and I will test it locally.
If nothing broken I will push new changes.
Thank you.